frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/lib/get-static-embed-setup-publish-handlers.ts

```diff
@@ -0,0 +1,87 @@
+import { countEmbeddingParameterOptions } from "metabase/embedding/lib/count-embedding-parameter-options";
+import type { StaticEmbedSetupPaneProps } from "metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane";
+import {
+  trackStaticEmbedDiscarded,
+  trackStaticEmbedPublished,
+  trackStaticEmbedUnpublished,
+} from "metabase/public/lib/analytics";
+import type {
+  EmbedResourceParameter,
+  EmbeddingParameters,
+} from "metabase/public/lib/types";
+
+import { getDefaultEmbeddingParams } from "./get-default-embedding-params";
+
+function convertResourceParametersToEmbeddingParams(
+  resourceParameters: EmbedResourceParameter[],
+) {
+  const embeddingParams: EmbeddingParameters = {};
+  for (const parameter of resourceParameters) {
+    embeddingParams[parameter.slug] = "disabled";
+  }
+
+  return embeddingParams;
+}
+
+export const getStaticEmbedSetupPublishHandlers = ({
+  resource,
+  resourceType,
+  resourceParameters,
+  onUpdateEnableEmbedding,
+  onUpdateEmbeddingParams,
+
+  embeddingParams,
+  setEmbeddingParams,
+
+  exampleDashboardId,
+
+  getAllParamsOnDiscard,
+}: StaticEmbedSetupPaneProps & {
+  embeddingParams: EmbeddingParameters;
+  setEmbeddingParams: (embeddingParams: EmbeddingParameters) => void;
+
+  exampleDashboardId: number | null;
+
+  getAllParamsOnDiscard: boolean;
```

Comment:
Can you also add a comment here on what this is for? It seems a bit out of place to me

Reply:
Removed the flag and enabled this behavior in all cases